.github/workflows/e2e_tests.yml

```
@@ -0,0 +1,183 @@
+          --health-start-period 60s
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v6
+        with:
+          ref: ${{ github.head_ref || github.ref }}
+          fetch-depth: 1
+
+      # Import Keycloak realm (service containers can't mount volumes, so we use the REST API)
+      - name: Wait for Keycloak to be ready
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2034:warning:2:1: i appears unused. Verify use (or export if used externally)

(shellcheck)